dev/Repeater/TestUI/Samples/ScaleAnimatedVerticalListDemo.xaml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Hosting;
+
+namespace MUXControlsTestApp.Samples
+{
+    public sealed partial class ScaleAnimatedVerticalListDemo : Page
+    {
+        public ScaleAnimatedVerticalListDemo()
+        {
+            this.InitializeComponent();
+            repeater.ItemsSource = Enumerable.Range(0, 100);
+            repeater.ElementPrepared += OnElementPrepared;
+        }
+
+        private void OnElementPrepared(Microsoft.UI.Xaml.Controls.ItemsRepeater sender, Microsoft.UI.Xaml.Controls.ItemsRepeaterElementPreparedEventArgs args)
+        {
+            var item = ElementCompositionPreview.GetElementVisual(args.Element);
+            var scrollProperties = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(sv);
+            var scaleExpresion = scrollProperties.Compositor.CreateExpressionAnimation();
+
+            scaleExpresion.SetReferenceParameter("sv", scrollProperties);
+            scaleExpresion.SetReferenceParameter("item", item);
+            scaleExpresion.SetScalarParameter("svHeight", 150);
+            scaleExpresion.SetScalarParameter("itemHeight", 30);
```

Comment:
Can we use args.Element.ActualHeight instead of hard-coded #'s here and for the others?

Reply:
When the element is prepared it is not measured yet, so the ActualHeight is not correct. It does not look like we can use ActualHeight in the expression either. I'll check if there is some other property from the visual i can use in the expression.